### ISSUE TITLE
Consider supporting hosting GraphQL SDL

### ISSUE THREAD

**Issue by oising** [Azure/API-Portal#12]
As the title says - it seems you only support REST/Swagger/OpenAPI for the moment. GraphQL would be a nice extension.

**Comment by azaslonov** [comment]
Hi @oising, yes, GraphQL support is coming around July-Aug.

**Comment by oising** [comment]
@azaslonov You guys should consider integrating a project like https://github.com/ChilliCream/hotchocolate instead of rolling your own stuff -- their gql playground equivalent (banana cake pop / bcp) is far more advanced than any others, notably it supports subscriptions.

**Comment by azaslonov** [comment]
@oising, yep, thanks for the suggestion, we're digging into this project as well. Looks very promising.

**Comment by oising** [comment]
Great -- we're putting it in production at the moment. The only thing I'd say is to avoid using the SDL/schema-first approach with it. It works, but it is behind the other approaches programmatic APIs in terms of flexibility and features (something they promise to look at in a near future release.)

**Comment by michaelstaib** [comment]
@azaslonov if you need any help or have any questions give us a shout. We are happy to help.

**Comment by azaslonov** [comment]
Thanks folks, we'll get back to then :)
